### ISSUE TITLE
Make bots auto-spawn/pre-selected as zombies on round start?

### ISSUE THREAD

**Issue by arthurgeron** [Dadido3/D3bot#13]
Hi!

I'm testing your addon, and I LOVE it, but there's just one issue.
Bots work perfectly, but won't spawn on their own.
I'd like them to be pre-selected as zombies before wave 1, I've seen that there are configurations that set numbers of zombies per human, but that seems to change anything for me.

**Comment by Dadido3** [comment]
Could be some incompatibility with the ZS you are running, which version do you use?
And what exactly is happening? Do they join the game, but don't spawn? It would help if you can give some more details.

**Comment by arthurgeron** [comment]
I'm running the latest version of Jetboom's zombiemod (github), also the latest version of D3Bot.
The zombies simply does not spawn on their own, if I add 1 bot it gets thrown into zombie class (which is good) and acts as it should, by that I mean it uses the D3Bot zombie AI. If I add more than 1 bot it goes to survival (I think that since I'm alone in the server it surpasses the limit of zombie bot per human).

Basically, everything seems to work except for the zombie spawn.. I've disabled server hibernation but still I don't know if that's the issue..
If you'd like to see that for yourself you can find the server under the name Redemption.

**Comment by Dadido3** [comment]
D3bot should be fully compatible with ZS:R.

Could it be that you spawn the bots manually, by using the `!bot` chat-command or the `bot` console-command?
If so, try `!botmod 2` (Or any other number) instead, this will put the desired bot count 2 zombies above what it would be normally. By using `!botmod 0` it will go back to normal, and `!botmod -100` will most likely kick all bots.

But i'm going to test it now, maybe there was a change in ZS:R a few days ago that broke stuff.

**Comment by arthurgeron** [comment]
I'll try that, thanks!
FYI server's ip is 173.208.184.234:27035

**Comment by Dadido3** [comment]
I was on your server. It seemed that bots spawn on every map, but they just run around randomly (Standard GMod behaviour for bots spawned with the `bot` console command). They are also named Bot01 and so on, so they aren't spawned by the newest version of the script.

Could it be that you still have an older version installed?

**Comment by arthurgeron** [comment]
I think I had added them manually, and that they stayed along, I'll try starting from scratch and see what happens.

**Comment by arthurgeron** [comment]
I've downloaded D3Bot and zombiemod again, 0 configs.. and nothing changed :/. I really don't know what is going on, I even removed any other addons that might be messing with it.

**Comment by Dadido3** [comment]
I can't really reproduce that error here.

Just some points and questions:
- What maps were the bots tested on? (Not all maps are supported out of the box, make sure to use a map which is in this [list of files](https://github.com/Dadido3/D3bot/tree/master/data/d3bot/navmesh/map)) There will be no bots spawning for any other map.
- Have you followed the installation in the `README.md` correctly? Maybe some point is confusing/not really clear, i could possibly improve that.
- Don't use the `bot` console command, instead change the desired bot count with `!botmod <number>` in the chat.
- I could help you on your server, when are you usually online there? You can reach me via the steam profile linked [in here](https://pastebin.com/KmCFjSCM)

**Comment by arthurgeron** [comment]
I fixed it, I had to navmesh files over to ` garrysmod/data/d3bot/navmesh/map`  I thought it wasn't necessary since I wasn't moving from an older version. Should've tried that sooner... hehe.. Thank you!
